tests/test_box_Box.py

```diff
@@ -485,6 +485,64 @@ def test_from_box(self):
         box7 = freud.box.Box.from_matrix(box.to_matrix())
         assert np.isclose(box.to_matrix(), box7.to_matrix()).all()
 
+    def test_standard_orthogonal_box(self):
+        box = freud.box.Box.from_box((1, 2, 3, 0, 0, 0))
+        Lx, Ly, Lz, alpha, beta, gamma = box.to_box_lengths_and_angles()
+        npt.assert_allclose(
+            (Lx, Ly, Lz, alpha, beta, gamma), (1, 2, 3, np.pi / 2, np.pi / 2, np.pi / 2)
+        )
+
+    def test_to_and_from_box_lengths_and_angles(self):
+        original_box_lengths_and_angles = (
+            1,
+            2,
+            3,
+            np.pi / 3,
+            np.pi / 2.25,
+            np.pi / 2.35,
+        )
+        box = freud.box.Box.from_box_lengths_and_angles(
+            *original_box_lengths_and_angles
+        )
+        lengths_and_angles_computed = box.to_box_lengths_and_angles()
+        assert np.allclose(lengths_and_angles_computed, original_box_lengths_and_angles)
+
+    def test_from_lattice_vectors_cubic_lattice(self):
```

Comment:
Should we parametrize these tests as cuboids? The math is pretty simple and it is (arguably) simpler:

```python
    @pytest.mark.parametrize(
        "cuboid_lengths",
        [
            [1, 1, 1],
            [1, 2, 3],
            [1, 1, 0],
            [4, 2, 0],
        ],
    )
    def test_from_lattice_vectors_cuboids(self, cuboid_lengths):
        lattice_vectors = np.diag(cuboid_lengths)
        unit_cell = freud.box.Box.from_lattice_vectors(lattice_vectors)

        exact_solution = [*cuboid_lengths, 0, 0, 0, 2 + (cuboid_lengths[-1] != 0)]
        freud_solution = [*unit_cell.to_dict().values()]

        assert np.allclose(freud_solution, exact_solution, atol=1e-15)
```